app.js

```diff
@@ -41,6 +41,23 @@ app.use(bodyParser.urlencoded({ extended: true }))
 var routes = require('./routes/main')
 app.use('/', routes)
 
+app.get('/bids', function(req, res, next){
+  res.render('bids',{ bids: [
```

Comment:
Same thing as kevin, now that you understand how to write endpoints , next time put the correct endpoint into the appropriate route